src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -194,7 +196,11 @@ const FacilitySearch = props => {
         <>
           <FacilityList {...facilityListProps} />
           {loadingMoreFacilities && loader()}
-          <button className="va-button-link" onClick={showMoreFacilities}>
+          <button
```

Comment:
Is there a reason for not using `va-button` here?

Reply:
@powellkerry , from the design perspective, UX wants this to appear as a link, and `va-button` does not provide that variant.

Reply:
Like @longmd said, design wanted this to look like a link and using the `va-button` does not provide a link like variant. So we'd either have to use it and add more custom styling to it (which feels like an anti-pattern), or use the generic button like I did above.